src/components/modal-flow/README.md

```diff
@@ -0,0 +1,42 @@
+# Modal Flow
+
+A component for modal flows. The layout will be inferred considering the flow variant.
+
+## Usage
+
+`<ModalFlow>` should be used along with `<ModalStep>`.
+
+```jsx
+import { ModalFlow } from '@nomios/web-uikit';
+
+<Modal isOpen>
+    <ModalFlow variant="simple" step="first">
+        <ModalStep id="first">
+            This is the first step.
+            <button onClick={ () => store.set({ stepId: 'second' }) } >Next step</button>
+        </ModalStep>
+        <ModalStep id="second">
+            This is the second step.
+            <button onClick={ () => store.set({ stepId: 'first' }) } >Next step</button>
+        </ModalStep>
+    </ModalFlow>
+</Modal>
+```
+
+There are four different layouts for the modal: `half`, `half-bordered`, `wide`, `full`. The layout applied depends on the chosen variant:
```

Comment:
I don't think we need this explanation.

Reply:
Don't you? I think it can help people to understand which layouts will be applied depending on the chosen variant.

Reply:
It might confuse people. I think this is a "inside detail" that readers of the readme do not need to be aware.

Reply:
Ok 👍

Reply:
Perhaps, we can just briefly say that there are different layouts according to the variant and the current step and link the diagram you posted above.